### PR TITLE
Create the Intuit Registration Page.

### PR DESCRIPTION
**Pull request by cod3smith**
#### What does this PR do?
- This PR contains the code for creating the `Intuit Registration Page`. 

#### How would you run this project?
- Clone the project.
- Run the `index.html` file on your browser.

### CODE REVIEW

File: .stickler.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+linters:
+  rubocop:
+    display_cop_names: true
+  csslint:
+  stylelint:
+  eslint:
+    config: './eslint.config'
```
Comment (lucasmazo32) [inline .stickler.yml]: - [x] Kindly enable the stickler for this repo.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                <span>mint</span>
+            </div>
+            <div class="icon_info">
+                <img class="icon" src="./images/quickbooks.svg" alt="Quickbooks Icon">
+            <span>quickbooks</span>
+            </div>
+            <div class="icon_info">
+                <img class="icon" src="./images/turbotax.svg" alt="Turbotax Icon">
+                <span>turbotax</span>
+            </div>
```
Comment (lucasmazo32) [inline index.html:45]: - [x] Kindly check the size of each text.

> Original
image

> Your page
image

**Review by lucasmazo32 — CHANGES_REQUESTED** [review]
# Changes required :wrench:

You are doing a great job so far, and the project is looking good. 

For the project to get approved kindly implement the changes below.

### Happy coding! :computer:

## Lucas Mazo

**Comment by berhanu-tarekegn** [comment]
@lucasmazo32 we fixed the linter issue and the font in the header section and also fixed issues raised  from morning session stand up meeting.

**Review by rammazzoti2000 — APPROVED** [review]
## Project Approved :rocket: :tada: 
Hello team great job, your project is approved, feel free to merge whenever you are ready.
###
Cheers and Happy coding! :clap: 

##### Code review by Alexandru Bangau

(Feel free to contact me on slack by the handle of alexandru if you have any additional question)</em>
### :heavy_check_mark: Status: Approved

**Pull request merged**